Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml

```diff
@@ -44,8 +44,8 @@
   id: ScarChest
   bodyPart: Chest
   markingCategory: Chest
-  speciesRestriction: [Human, Dwarf, Felinid, Oni, Feroxi] #Einstein Engines - Felinid, Oni - DeltaV - Feroxi
-  sexRestriction: [Male] # DeltaV: Splitting the scars and tattoos
+  speciesRestriction: [Human, Dwarf, Felinid, Oni, Feroxi, Vulpkanin] #Einstein Engines - Felinid-Oni - DeltaV - Feroxi-Vulpkanin
```

Comment:
idk why these had EE comments to begin with felinid and oni were stolen from nyano do the same for these